wafw00f/plugins/baidu.py

```diff
@@ -9,7 +9,10 @@
 
 def is_waf(self):
     schemes = [
-        self.matchHeader(('Server', r'Yunjiasu(.+)?'))
+
+        # just added another signature
+        self.matchHeader(('Server', r'Yunjiasu(.+)?')),
+        self.matchHeader(('Server', 'yunjiasu-nginx'))
```

Comment:
This too seems to be a bit redundant, I think the regex `Yunjiasu(.+)?` matches `yunjiasu-nginx` as well when grepped case insensitive.